homeassistant/components/deconz/sensor.py

```diff
@@ -97,11 +97,13 @@ class DeconzSensor(DeconzDevice):
     """Representation of a deCONZ sensor."""
 
     @callback
-    def async_update_callback(self, force_update=False):
+    def async_update_callback(self, force_update=False, ignore_update=False):
         """Update the sensor's state."""
-        changed = set(self._device.changed_keys)
+        if ignore_update:
```

Comment:
This seems kinda weird. Wouldn't it be possible to just not call `async_update_callback` if `ignore_update` is True ?

Reply:
My thought was to not have that logic in the library, but you think that would be better?